relnotes/stream-stuff.deprecation.md

```diff
@@ -1,3 +1,5 @@
-* `ocean.io.stream.Digester`, `ocean.io.stream.Endian`, `ocean.io.stream.Greedy`, `ocean.io.stream.Map`, `ocean.io.stream.Typed`, `ocean.io.stream.Utf`
+## Deprecate unused modules
```

Comment:
Maybe ```Deprecate unused `io` modules```? This title as it is kind of overlaps with the whole section title, it doesn't provide any further useful information.

Actually for deprecations it might even make sense to keep the old list format, as usually what you need to know is which modules were deprecated and the grouping of different modules based on how the deprecations were committed is not very important/useful. :thinking:

Reply:
Not all deprecated modules are unused though

Reply:
>Maybe Deprecate unused `io` modules?

:+1:

>Actually for deprecations it might even make sense to keep the old list format

I thought that too, at first, but then noticed that there are different types of deprecation. Despite what we see in this PR, the most common type is actually "Replace X with Y", I'd say, which makes a pretty useful quick summary.

Reply:
Updated.